session/session.go

```diff
@@ -2644,6 +2644,7 @@ func CreateSession4TestWithOpt(store kv.Storage, opt *Opt) (Session, error) {
 		// initialize session variables for test.
 		s.GetSessionVars().InitChunkSize = 2
 		s.GetSessionVars().MaxChunkSize = 32
+		s.GetSessionVars().EnablePaging = true
```

Comment:
It should be `sessionctx.variable.DefTiDBEnablePaging`

Otherwise if we reset the default value, we may miss the chage here.